lib/datasource/conan/index.spec.ts

```diff
@@ -7,26 +7,29 @@ import { defaultRegistryUrl } from './common';
 import { ConanDatasource } from '.';
 
 const pocoJson = Fixtures.get('poco.json');
+const pocoYamlGitHubContent = Fixtures.get('poco.yaml.json');
 const malformedJson = Fixtures.get('malformed.json');
 const fakeJson = Fixtures.get('fake.json');
 const datasource = ConanDatasource.id;
 
+const nonDefaultRegistryUrl = 'https://not.conan.io/';
```

Comment:
It was easiest to keep the existing tests but change registry URL used to avoid the conan center github lookup